### DISCUSSION TITLE
Install TRMM without meshcentral

### DISCUSSION THREAD

**Discussion by pggsadmin** [amidaware/tacticalrmm#1147] [Q&A] [answered]
Is it possible to install Tactical RMM without meshcentral. Because I don't need allow remote access to any one.
I require only scripting and othe patch update only.

**Comment by dinger1986** [comment]
You could take it out of the install script

**Comment by wh1te909** [comment, marked as answer]
At the moment not really, taking it out of install might mess things up after install since certain database fields would be missing. I would recommend doing a normal install and then just disable meshcentral/mongo with the following:
```
sudo systemctl disable --now meshcentral mongod
```
And then when installing an agent, make sure to pass the `-nomesh` flag to the installer: https://docs.tacticalrmm.com/install_agent/#optional-installer-args

**Reply by silversword411** [reply]
You think running `upgrade.sh` is going to have other things that will require fixing every time as well?

Thinking I'm going make a dedicated no mesh docs page on this topic

**Reply by pggsadmin** [reply]
@silversword411  Can you incorporate a dedicated no mesh doc ?

**Reply by silversword411** [reply]
> @silversword411 Can you incorporate a dedicated no mesh doc ?

https://docs.tacticalrmm.com/mesh_integration/#using-tactical-rmm-without-any-meshcentral

**Reply by dinger1986** [reply]
Could also uninstall 

Might need to remove /etc/nginx/sites-enabled/meshcentral.conf or sites-available so it doesn't mess up nginx.

**Comment by VINISHVV** [comment]
Hi,
Any progress
